setup.py

```diff
@@ -31,8 +32,8 @@ def read(fname):
     ],
     install_requires=[
         "PyYAML==3.11",
-        "upseto",
     ],
     include_package_data=True,
     data_files=data_files,
+    scripts=['sh/solvent']
```

Comment:
The uninstall recipe won't work on Ubuntu since this is placed under "/usr/local/lib". Fix this (so that the makefile attempts to remove the file from /usr/local/lib as well). And test it on Ubuntu.